vcbuild.bat

```diff
@@ -778,6 +774,7 @@ for /D %%D IN (doc\*) do (
   )
 )
 %node_exe% tools\lint-md\lint-md.mjs %lint_md_files%
+%npx_exe% --prefix tools\doc api-docs-tooling -i doc\api\*.md
```

Comment:
Why did we start using NPX? I don't think we should depend on npm, anymore than we should.

Reply:
> I don't think we should depend on npm, anymore than we should.

Genuinely curious: Is there a technical reason for that or is this about package manager neutrality? (I definitely don't want to have the package manager neutrality conversation here as it would be very off-topic. I just want to know if your reasons are philosophical or technical. I respect your request for change either way.)

Reply:
Agree with Trott. Also the package is already being installed. We are using npx because "node --run" doesnt support running from a package.json in a different folder.

We tried numerous workarounds, had to use npx.

Reply:
Please don't make this a requirement. But I agree it would definitely be a good issue to be worked on :)

Reply:
> Please don't make this a requirement. But I agree it would definitely be a good issue to be worked on :)

I can just implement this if no one is interested in working on, if you give enough time. :-)

Reply:
We're not in a hurry anyways.

Reply:
> We're not in a hurry anyways.

Can someone open an issue for this and assign me?

Reply:
#57489

Reply:
The current old tooling uses npm, but doesn't use npx. I prefer to wait, but I'll not block.

Reply:
Appreciate ya! I do believe we are for now blocked anyways and we need to handle @aduh95's concern regarding `make docserve` I guess?